servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -835,4 +837,10 @@ default StreamingHttpClientFilter create(FilterableStreamingHttpClient client) {
             return create(client, null, null);
         }
     }
+
+    private static <ResolvedAddress> HttpLoadBalancerFactory<ResolvedAddress> defaultLoadBalancer() {
+        return DefaultHttpLoadBalancerFactory.Builder.from(
+                RoundRobinLoadBalancers.<ResolvedAddress, FilterableStreamingHttpLoadBalancedConnection>builder(
+                                DefaultHttpLoadBalancerFactory.class.getSimpleName()).build()).build();
+    }
```

Comment:
This exists because we no longer have the `fromDefaults()` method on the `DefaultHttpLoadBalancerFactory` as the RR isn't in scope in that package and it doesn't make sense to define the default load balancer in the API package.